packages/protocol/contracts/common/ICrossChainSync.sol

```diff
@@ -14,16 +14,24 @@ pragma solidity ^0.8.20;
 /// both chains remain consistent and can be cross-referenced with integrity.
 interface ICrossChainSync {
     struct Snippet {
+        uint64 remoteBlockId;
+        uint64 syncedInBlock;
         bytes32 blockHash;
         bytes32 signalRoot;
     }
 
     /// @dev Emitted when a block has been synced across chains.
-    /// @param srcHeight The height (block id_ that was synced.
+    /// @param remoteBlockId The ID of the remote block whose block hash and
+    /// signal root are synced.
+    /// @param syncedInBlock The ID of this chain's block where the sync
+    /// happened.
     /// @param blockHash The hash of the synced block.
     /// @param signalRoot The root hash representing cross-chain signals.
     event CrossChainSynced(
-        uint64 indexed srcHeight, bytes32 blockHash, bytes32 signalRoot
+        uint64 indexed remoteBlockId,
+        uint64 syncedInBlock,
+        bytes32 blockHash,
+        bytes32 signalRoot
```

Comment:
Nitpick: a bit of a strange order of these variables, `syncedInBlock` of the current chain is mixed between variable of the source chain. Also the there's a `remote` in front of `blockId`, but not in front of `blockHash` and `signalRoot` which are also `remote` if I understand correctly. So something like would be more clear I think:
```
uint64 syncedInBlock,
uint64 indexed blockId,
bytes32 blockHash,
bytes32 signalRoot
```

Reply:
Please take a look at this PR: https://github.com/taikoxyz/taiko-mono/pull/15202